Assets/MixedRealityToolkit/Inspectors/Data/DefaultToolboxItems.json

```diff
@@ -0,0 +1,181 @@
+{
+  "Categories": [
+    {
+      "CategoryName": "Buttons",
+      "Items": [
+        {
+          "Name": "HoloLens2 Button",
+          "RelativeAssetPath": "Features\\UX\\Interactable\\Prefabs\\PressableButtonHoloLens2.prefab",
+          "AssetModule": 5,
+          "IconModule": 1,
+          "RelativeIconPath": "Inspectors\\Icons\\MRTK_Button_Icon.png",
```

Comment:
Rather than add a bunch of more icons to another area of the toolkit, consider reusing existing textures in the standard assets, or have a refined set within the Standard assets.  Let's not create more textures around the toolkit outside of the "assets" folder

Reply:
Moving icons to the Standard assets folder area is fine...but I don't see how I can re-use anything in the standard assets? Nothing is pre-existing that demonstrates each UX component....except for in the documentation folder

As I said though in my PR description, the icons definitely need some work. They are my quick hack to demonstrate something. They have not been optimized for consistency or size